composer.json

```diff
@@ -39,7 +39,7 @@
 		"classmap-authoritative": true,
 		"autoloader-suffix": "SuspiciousLogin",
 		"platform": {
-			"php": "7.4"
+			"php": "8.0"
```

Comment:
sneaky sneaky. this implicitly makes Dependabot bump to 7.4 incompatible dependencies. let's do this together with the update of the min php in info.xml

Reply:
https://github.com/nextcloud/suspicious_login/issues/807